src/components/CertificateDropZone/CertificateDropZone.stories.tsx

```diff
@@ -8,11 +8,38 @@ import {
   whenDocumentIssuerIdentityInvalidDnsTxt,
   whenDocumentHashInvalidAndNotIssued,
 } from "../../test/fixture/verifier-responses";
-import { VerificationFragment } from "@govtechsg/oa-verify";
 import { Provider } from "react-redux";
 import { configureStore } from "../../store";
+import { states } from "../../reducers/certificate";
 
-const store = configureStore();
+const RenderWithStore = ({ children, ...props }: any) => {
+  const {
+    raw,
+    rawModified,
+    verificationStatus,
+    verificationPending,
```

Comment:
why need both verificationPending when have verificationStatus.isVerificationPending?

Reply:
from my understanding verificationStatus is the verified fragments, which will run through interpret fragments to get the 3 ticks. so shouldn't have verificationStatus.isVerificationPending